packages/react-components/src/organisms/ProfileSkills.tsx

```diff
@@ -11,12 +13,21 @@ type ProfileSkillsProps = {
 const ProfileSkills: React.FC<ProfileSkillsProps> = ({
   skills = [],
   skillsDescription,
-}) => (
-  <Card>
-    <Headline2 styleAsHeading={3}>Expertise and Resources</Headline2>
-    {skillsDescription ? <Paragraph>{skillsDescription}</Paragraph> : null}
-    <TagList tags={skills} />
-  </Card>
-);
+}) => {
+  const { isOwnProfile } = useContext(UserProfileContext);
+  return skills.length || isOwnProfile ? (
```

Comment:
What if `!skills.length && !isOwnProfile && skillsDescription`, wouldn't this incorrectly hide the whole thing?

Reply:
Like the other place i have preserved the behaviour. Without a requirement in the ticket i am reluctant to change